requirements.txt

```diff
@@ -1,6 +1,5 @@
 numpy <2.0  # needed for older Torch
 torch <2.3  # fixme: freeze until AMP compatibility is resolved
 lightning >=2.0.0
-hivemind >=1.1.0, <=1.1.10.post2; sys_platform == 'linux'
-
-pydantic <2.0.0  # fixme: lift when resolved
+hivemind @ git+https://github.com/learning-at-home/hivemind.git@213bff98a62accb91f254e2afdccbf1d69ebdea9
```

Comment:
lets wait till this is fully released

Reply:
@Vectorrent was it released?

Reply:
I just pinged a maintainer, and was told we can expect a release by end of week.

Reply:
I know that PyPI restrict only published versions not source or commit...